service/org.nemomobile.BtrfsBalancer.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Interface=org.nemomobile.BtrfsBalancer
+Name=org.nemomobile.BtrfsBalancer
+Exec=/usr/sbin/btrfs-balancer server
+User=root
```

Comment:
In the future, however, we might want to balance right after slipstreaming, which would be quite early in the boot process, I assume.